content/how-to/fabric/fabric-ticketing/concept-based-article-template.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
you can delete this file

Reply:
Deleted - concept-based-article-template.md